src/compiler/builderPublic.ts

```diff
@@ -34,9 +39,9 @@ namespace ts {
         /*@internal*/
         getState(): ReusableBuilderProgramState;
         /*@internal*/
-        backupState(): void;
+        backupEmitState(): void;

```

Comment:
Where are these called?  It seems odd that there'd be no parameters or return value.

Reply:
Its called in [tsbuild](https://github.com/microsoft/TypeScript/pull/48784/files#diff-0558af289c76f66611354da63a14152fe5cd3a72b3c3e59eb880808efc2df442L925)